tests/users/test_api_user.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+from flask import json
+
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.base_test_case import BaseTestCase
+from tests.test_data import user1
+
+
+class TestUserApi(BaseTestCase):
```

Comment:
```suggestion
class TestResendEmailApi(BaseTestCase):
```
Can you also change the name of the fil to reflect this, since its too broad of a name, when this is actually related to Resend Email API